fake_spectra/singleabs.h

```diff
@@ -25,6 +25,23 @@ inline double sph_kernel(const double q)
         return norm*(2*pow(1.- q,3));
 }
 
+/* The normalized quintic kernel from Price 2011: arxiv 1012.1885 , eq. 8
+ * We define the support over h < 1, rather than h < 3 used there.
+ * This changes the values of the constants, but the kernel value is the same.*/
+inline double sph_kernel(const double q)
+{
+    const double norm = 9./40/M_PI;
+    if (q >= 1)
+        return 0;
+    if(q < (1./3))
+        return norm*6*(11-90*q*q+405*q*q*q*q-405*q*q*q*q*q);
+    if(q < (2/3) && q >= (1/3))
```

Comment:
Can you make these 2./3 and 1./3 so the compiler doesn't decide to get clever and return 0?